.github/CODEOWNERS

```
@@ -98,10 +98,10 @@ clang/tools/clang-offload-deps/ @sndmitriev @mlychkov @AlexeySachkov
 clang/tools/clang-offload-extract/ @sndmitriev @mlychkov @AlexeySachkov
 
 # Explicit SIMD
-SYCLLowerIR/ @kbobrovs @DenisBakhvalov @kychendev
-esimd/ @kbobrovs @DenisBakhvalov @kychendev
-sycl/include/sycl/ext/intel/experimental/esimd.hpp @kbobrovs @DenisBakhvalov @kychendev
-sycl/doc/extensions/ExplicitSIMD/ @kbobrovs @DenisBakhvalov @kychendev
+SYCLLowerIR/ @kbobrovs @sndmitriev @kychendev
```

I am in process of doing some big patch for SYCLLowerIR. Maybe add my name to owner there too?
```suggestion
SYCLLowerIR/ @kbobrovs @sndmitriev @kychendev @v-klochkov 
```